src/test/scala/firrtlTests/RenameMapSpec.scala

```diff
@@ -71,4 +75,55 @@ class RenameMapSpec extends FirrtlFlatSpec {
     renames.rename(foo, bar)
     renames.get(foo) should be (Some(Seq(barB)))
   }
+
+  it should "rename modules if their circuit is renamed" in {
+    val renames = RenameMap()
+    renames.rename(cir, cir2)
+    renames.get(modA) should be (Some(Seq(modA2)))
```

Comment:
Shouldn't you also check that modB renamed?

Reply:
In this case, the only thing in the rename map is the circuit rename. Thus checking for modA and modB is identical.